### PR TITLE
Add 2022 Q3 updates

### PR DESCRIPTION
**Pull request by nathanshammah**
Add quarterly update blog post for 2022 Q3.

### CODE REVIEW

File: posts/_markdown/2022_Q3.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+---
+
+Dear Unitary Fund community,
+
+We’re excited to share our quarterly update and highlight some of what UF’s been involved with so far this year. We keep growing the Mitiq project and Metriq.info platform with a wider community. We hosted two fantastic interns over the Summer, Maria and Karen, teaming up with Qubit by Qubit. We have awarded 11 grants to amazing projects this quarter, spanning from the first Rust project supported by UF to an outreach program for high-school students. Finally, we’ve released the Quantum Open Source Software Survey and collected over 1000 responses: Thank you all for your continued support of the open-source quantum ecosystem!
+
+
+Make sure to follow our
+[[Discord]{.ul}](https://discord.com/invite/JqVGmpkP96),
+[[Twitter]{.ul}](https://twitter.com/unitaryfund),[[LinkedIn]{.ul}](https://www.linkedin.com/company/unitary-fund/),
```
Comment (natestemen) [inline posts/_markdown/2022_Q3.md:14]: Needs a space after the twitter link, before linkedin.

File: posts/_markdown/2022_Q3.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+        -   Version 0.19 expanded support to the most recent versions of
+             Python: 3.8, 3.9 and 3.10, and is now compatible with
+             Numpy 1.21.6. We have also added new tools for applying
+             learning-based PEC, and introduced a function for learning
+             depolarizing noise representations from Clifford circuit
+             data.
+
+        -   Read more in our updated
+             [API-doc](https://mitiq.readthedocs.io/en/latest/apidoc.html#module-mitiq.pec.representations.biased_noise).
```
Comment (natestemen) [inline posts/_markdown/2022_Q3.md:44]:
```suggestion
        -   Read more about learning-based PEC in our updated
             [API-doc](https://mitiq.readthedocs.io/en/latest/apidoc.html#module-mitiq.pec.representations.biased_noise).
```

File: posts/_markdown/2022_Q3.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+             [\@amirebrahimi](https://github.com/amirebrahimi) and
+             [\@isaac-gs](https://github.com/isaac-gs)!
+
+    -   Talks:
+
+        -   September 22nd, 2022 - [IEEE Quantum
+             Week](https://qce.quantum.ieee.org/2022/), [Workshop on
+             quantum
+             IR](https://qce.quantum.ieee.org/2022/workshops-program/#alexandermccaskey),
+             *\"Mitiq: A cross-platform error mitigation toolkit\"*.
```
Comment (natestemen) [inline posts/_markdown/2022_Q3.md:57]: Do quotes `"` need to be escaped?

File: posts/_markdown/2022_Q3.md [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+             Conference](https://qa.eaifr.org/), *\"Quantum Error
+             Mitigation: An open-source software approach\"*.
+             \@nathanshammah
+
+        -   August 15th, 2022 - [WOMANIUM
+             QUANTUM](https://www.youtube.com/watch?v=3LAHjmSuyO8&list=PL-VMs2BCTI_lklMMfY4iMdETT19rgZe5o)
+             \@Misty-W
+
+    -   The [[Mitiq whitepaper]{.ul}](https://arxiv.org/abs/2009.04417)
+         was published in *Quantum **6**, 774 (2022)*
```
Comment (natestemen) [inline posts/_markdown/2022_Q3.md:70]: Should we use the quantum URL for this?

https://quantum-journal.org/papers/q-2022-08-11-774/

File: posts/_markdown/2022_Q3.md [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+
+-   [**[Metriq]{.ul}**](https://metriq.info/)
+
+    -   New Releases:
+         [[v.0.6.0]{.ul}](https://github.com/unitaryfund/metriq-app/releases/tag/v0.6.0),
+         [[v.0.6.1]{.ul}](https://github.com/unitaryfund/metriq-app/releases/tag/v0.6.1)
+
+        -   Version 0.6.0 focuses on UX upgrades, including new member
+             self-curation by granular subscription. Single-page
```
Comment (natestemen) [inline posts/_markdown/2022_Q3.md:79]: I don't know what this sentence means. What is "new member self-curation by granular subscription"?

File: posts/_markdown/2022_Q3.md [hunk 6]
```diff
@@ -0,0 +1,263 @@
...
+
+    -   You can submit results on existing benchmarks or propose new
+         benchmarking tasks to the community. Visit Metriq.info and the
+         [[\#metriq]{.ul}](https://discord.com/channels/764231928676089909/818208195612639304)
+         channel on the Unitary Fund Discord to learn more!
+
+-   **New Papers Published by Unitary Fund**
+
+    -   Testing platform-independent quantum error mitigation on noisy
+         quantum computers.
```
Comment (natestemen) [inline posts/_markdown/2022_Q3.md:106]: Needs link. https://arxiv.org/abs/2210.07194

**Review by natestemen — COMMENTED** [review]
Since we are including things in this blog post that happened into Q4, should we include the mitiq 0.20 release?

**Comment by nathanshammah** [comment]
@natestemen "No" referred to the question about citing Q4 releases.

**Review by natestemen — APPROVED** [review]

**Review by willzeng — APPROVED** [review]

File: blog.html [hunk 7]
```diff
@@ -72,6 +72,10 @@
               </p>
               <section class="light">
                 <ul style="list-style-type: none">
+                  <li class="leading-block">
+                    <span class="blog-post-date">9 November, 2022</span>
```
Comment (nathanshammah) [inline blog.html:76]:
```suggestion
                    <span class="blog-post-date">11 November, 2022</span>
```

File: posts/2022_Q3.html [hunk 8]
```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<html lang="en">
+<script src="https://kit.fontawesome.com/22ffc91bdd.js" crossorigin="anonymous"></script>
+
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8">
+    <meta name="date" content='2022-11-09'>
```
Comment (nathanshammah) [inline posts/2022_Q3.html:9]:
```suggestion
    <meta name="date" content='2022-11-11'>
```

**Pull request merged**